src/namespace.js

```diff
@@ -0,0 +1,81 @@
+
+export const namespaceSeparator = '/'
```

Comment:
I used `/` here for Vuex compatibility. In our case a dot-path could be reasonable as well (maybe even better)